extensions.json

```
@@ -154,6 +154,11 @@
       "version": "0.5.1",
       "checkout": "0.5.1"
     },
+    {
+      "id": "bierner.docs-view",
+      "repository": "https://github.com/mattbierner/vscode-docs-view.git",
```

Minor personal preference (sorry 😅):

```suggestion
      "repository": "https://github.com/mattbierner/vscode-docs-view",
```